### ISSUE TITLE
UI tests fail during PVC creation with TimeoutException: Message: Failed to find the element

### ISSUE THREAD

**Issue by sidhant-agrawal** [red-hat-storage/ocs-ci#10248]
UI tests in `test_pvc_ui.py` fail during PVC creation with the following error:

```
E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (css selector,input[value='ReadWriteOnce'])
```

```
E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (css selector,input[value='ReadWriteMany'])
```

RP links for 4.16 runs: 
 - https://url.corp.redhat.com/90bcd2e
 - https://url.corp.redhat.com/78bc0e2

The failure seems to be due to a recent UI change. The access mode selection has been changed from radio input to a menu dropdown. The tests are unable to locate the expected elements in the new UI.

**Comment by DanielOsypenko** [comment]
tests are failing on ROSA HCP. We need to address this issue to qualify new platform type. Please address it and pull the fix to 4.16
https://url.corp.redhat.com/de7a206

**Comment by nehaberry** [comment]
> tests are failing on ROSA HCP. We need to address this issue to qualify new platform type. Please address it and pull the fix to 4.16 https://url.corp.redhat.com/de7a206

@ebondare did we resolve on this too  with the PR you submitted?
